docs/architecture/solana/migration/governance.mdx

```diff
@@ -27,8 +27,8 @@ wallet, and vote.
 ## Governance Changes
 
 The major change with Solana governance is that more of the Helium Network can be governed by the
-Helium DAO and its subDAOS. Everything from Program (smart contract) changes to approving Makers can
-now be controlled by the Helium DAO and its subDAOs.
+Helium DAO and its Subnetworks. Everything from Program (smart contract) changes to approving Makers can
```

Comment:
'Helium DAO' should be reworked here too.

Reply:
Fixed